dateparser_data/settings.py

```diff
@@ -29,4 +29,10 @@
     # Other settings
     'RETURN_TIME_AS_PERIOD': False,
     'PARSERS': default_parsers,
+
+    'DEFAULT_LANGUAGES': ["en"],
```

Comment:
Is this a backwards incompatible change?

Reply:
Yes, I believe its backward incompatible.

Reply:
Ok, I think we have the following options here:

- make this change - but we should be really use it's for the better in most cases
- don't make this change but highlight in the docs that for custom lang detection you should probably adjust this
- somehow do that only for custom lang detection by default

what do you think folks?

Reply:
Sorry - actually it's not a breaking change as this setting is used only for custom language detection.